lib/edge/kotlin/build-aar.sh

```
@@ -0,0 +1,181 @@
+if ! command -v cargo-ndk >/dev/null 2>&1; then
+    echo "ERROR: cargo-ndk not found. Install with: cargo install cargo-ndk" >&2
+    exit 1
+fi
+
+if [ -z "${ANDROID_NDK_HOME:-}" ]; then
+    if [ -z "${ANDROID_HOME:-}" ]; then
+        echo "ERROR: ANDROID_NDK_HOME or ANDROID_HOME must be set." >&2
+        echo "  export ANDROID_NDK_HOME=/path/to/ndk" >&2
+        exit 1
+    fi
+    NDK_DIR=$(ls -d "$ANDROID_HOME/ndk/"* 2>/dev/null | sort -V | tail -1 || true)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)